### DISCUSSION TITLE
Auro-icons/badge in auro-select

### DISCUSSION THREAD

**Discussion by eholloway1** [AlaskaAirlines/WC-Generator#544] [Announcements]
Hi,
 
I am trying to make a auro-select menu of cards-on-file, but selected card appears with the raw html for the auro elements.
 
![image](https://github.com/AlaskaAirlines/WC-Generator/assets/12372675/18c77313-ac2c-4734-b184-eab8e18a3122)

Even on the auro site, they have this problem with icons in a select menu.
 
![image](https://github.com/AlaskaAirlines/WC-Generator/assets/12372675/657b36b3-7a39-49af-8d04-e2d96c46dd45)

Has anyone tackled this issue before, or know someone who can help? Thanks in advanced!

**Comment by eholloway1** [comment]
I found this inspecting the elements of the dropdown:
![image](https://github.com/AlaskaAirlines/WC-Generator/assets/12372675/47048bd7-f908-4da0-a07b-f702b14235b8)

It looks like the selected menuoption is printed in a span with a property 'valuestr' set to true. Maybe this causes whatever is encapsulated in that span to be rendered as text. Not sure how to fix this.

**Comment by jordanjones243** [comment]
@eholloway1 Upon investigating the code, the value selected is not only being wrapped in a `span` element, but it is being created as a text node and being appended to that `span` element. This is a bug on our end and an issue can be submitted to get this fixed here:
https://github.com/AlaskaAirlines/auro-select/issues
